### ISSUE TITLE
Explorer auto refresh intervall

### ISSUE THREAD

**Issue by calipee** [vscode-kubernetes-tools/vscode-kubernetes-tools#1131]
I'd really love if someone could add an auto refresh option for the explorer like for the docker extension `docker.explorerRefreshInterval`. 

Sometimes it can be annoying to refresh the explorer manually.

**Comment by znorris** [comment]
Agreed. Not sure why it's a good user experience to have me clicking that refresh button in the top right of the pane all the time.

**Comment by gannaramu** [comment]
+1 this would be a really good feature to have.
